packages/gatsby-cli/src/reporter/types.d.ts

```diff
@@ -0,0 +1,29 @@
+export interface ActivityTracker {
+  start(): Function
```

Comment:
I think all these need terminators at the end `;` satisfies a common TSLint and prettier default config

Reply:
They were removed after running the current `prettierlint --fix` and was throwing linting errors when they were present. I'm unsure how to follow (as I saw errors from webstorm mentioning this as well)